lib/system/memalloc.nim

```diff
@@ -375,7 +375,7 @@ when hasAlloc and not defined(js):
       else:
         dealloc(p -! offset)
 
-  proc alignedRealloc(p: pointer, oldSize, newSize, align: Natural): pointer =
```

Comment:
don't export those from system, use a dedicated module under `std/` and use import/re-export mechanism to avoid duplication